classes.py

```diff
@@ -1 +1,34 @@
-# some_code
+class Rna(str):
+    sequence = ''
+    
+    def __init__(self, sequence):
+        if isinstance(self.sequence, str):
+            self.sequence = sequence.upper()
+        else:
+            raise TypeError
+            
+    def gc(self):
+        gc_count = self.sequence.count('C') + self.sequence.count('G')
+        gc_content = gc_count * 100 / len(self.sequence)
+        return round(gc_content, 2)
+
+    def reverse_complement(self):
+        complement_sequence = self.sequence.replace('A', 'u').replace('U', 'a').replace('G', 'c').replace('C', 'g')
+        complement_sequence = complement_sequence.upper()
```

Comment:
you can combine this line and the next one.

Reply:
Done

Reply:
Why do we need the second line at all?